cmp/grid/renderers/MultiFieldRenderer.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {multiFieldCell} from '../impl/MultiFieldCell';
+
+/**
+ * A grid elementRendererFn that renders a collection of additional sub fields in a row beneath the main column field.
+ *
+ * Requires the column to also specify a multiFieldRendererCfg, with the following params:
```

Comment:
Describe what the SubFields are in *this* public file.  Don't want to refer user to that private impl class...  (which we may want to remove anyway, as mentioned)